flocker/acceptance/testtools.py

```diff
@@ -89,6 +90,13 @@
 SOCKET_TIMEOUT_FOR_POLLING = 2.0
 
 
+class FailureToUpgrade(Exception):
+    """
+    Exception raised to indicate a failure to install a new version of Flocker.
+    """
+    pass
```

Comment:
Removed

I don't understand when to or not to use pass :(

Would you mind teaching me in the form of a suggested edit to https://docs.google.com/document/d/1WyPosnJuIyAdPHnI5E0D5QYzcwhyBqVYIEZoHeh3ndQ/edit ?  Might not be the best use of your time though... maybe I'll look it up and add a section later.